client/submitAssignment.js

```diff
@@ -20,7 +24,11 @@ window.onload = function() {
     });
 };
 
-// Uploads the form data to server.
+/**
+ * Validates assignment tarball file and sends a POST request to server
+ * for uploading the tarball and other assignment related details.
+ * It logs the success and error status of the operation.
```

Comment:
we don't need to mention this.

Reply:
Are you talking about the last line about logging success and error message?

Reply:
Yes.